Makefile

```diff
@@ -53,3 +53,8 @@ run-quality-test: clean ## Run quality test.
 	$(TOX) pycodestyle ./eox_tagging
 	$(TOX) pylint ./eox_tagging --rcfile=./setup.cfg
 	$(TOX) isort --check-only --recursive --diff ./eox_tagging
+
+build-docs:
+	pip install sphinx
```

Comment:
@mariajgrimaldi I think that we should add a requirements/docs.in and a requirements/docs.txt files with the packages needed to generate the documentation, and use that here. We also would need to update the `upgrade` target to generate the  requirements/docs.txt file

Reply:
That would be better! 
The upgrade part is something like: 
$(PIP_COMPILE) -o requirements/docs.txt requirements/docs.in right?

Reply:
yes